staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -285,8 +277,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	AggregatedDiscoveryEndpoint: {Default: true, PreRelease: featuregate.Beta},
 
-	AdmissionWebhookMatchConditions: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Let's update to 

`AdmissionWebhookMatchConditions: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.33` 

Instead to avoid breaking consumers that are passing `--feature-gates=AdmissionWebhookMatchConditions=true`

Reply:
done